common/test/acceptance/tests/studio/test_studio_help.py

```diff
@@ -47,3 +63,357 @@ def test_studio_help_links(self):
+            {
+                'element_css': NAV_HELP_CSS,
+                'text': 'Help',
+                'url': 'http://edx.readthedocs.org/projects/edx-partner-course-staff/'
+                       'en/latest/getting_started/get_started.html',
+                'type': 'library tab'
+            }
+        ]
+        for expected_link in expected_links:
+            self.auth_page.visit()
+            self.dashboard_page.visit()
+            if expected_link['type'] == 'course':
+                self.dashboard_page.click_new_course_button()
+            elif expected_link['type'] == 'library':
+                self.dashboard_page.click_new_library()
+            else:
+                self.assertTrue(self.dashboard_page.has_new_library_button)
+                click_css(self.dashboard_page, '#course-index-tabs .libraries-tab', 0, False)
+
+            element_css = expected_link['element_css']
+            # Assert that link on DOM element are same as expected.
+            actual_link = get_element(self.dashboard_page, element_css)
+            assert_links(self, expected_link, actual_link)
+            # Click the help link and assert that correct link is opened.
+            click_css(self.dashboard_page, element_css, 0, False)
+            assert_help_is_open(self, expected_link['url'])
+
+
+class LibraryHelpTest(StudioLibraryTest):
+    """
+    Test help links on a Library page.
+    """
+    def setUp(self):
+        super(LibraryHelpTest, self).setUp()
+        self.library_page = LibraryPage(self.browser, self.library_key)
+        self.library_user_page = LibraryUsersPage(self.browser, self.library_key)
+
+    def test_library_content_and_library_user_access_help(self):
+        """
+        Scenario: Help links in navigation bar and sidebar are working on content
+        library page(click a library on the Library list page). Also Help link in
+        navigation bar is working on 'User Access' settings page of library.
+
+        Given that I am on the content library page(click a library on the Library list page).
+        And I want help
+        And I click the 'Help' in the navigation bar
+        Then Help link should open.
+        And help url should contain 'course/components/libraries.html'
+
+        Given that I am on the content library page(click a library on the Library list page).
+        And I want help
+        And I click the 'Learn more about content libraries' in the sidebar links
+        Then Help link should open.
+        And help url should contain 'course/components/libraries.html'
+
+        Given that I am on the 'User Access' settings page of library.
+        And I want help
+        And I click the 'Help' in the navigation bar
+        Then Help link should open.
+        And help url should contain
+        'creating_content/libraries.html#give-other-users-access-to-your-library'
+        """
+        expected_links = [
+            {
+                'element_css': NAV_HELP_CSS,
```

Comment:
Are the tests using the element_css and label to recognize the link that we're testing?

Since we're frequently testing links from the "help" button on the navigation bar, could this be defined in one place and reused for each test class? I'm wondering how hard it would be to update the tests if someone relabels  the button or changes the class name.